include/modules/sway/language.hpp

```diff
@@ -32,6 +32,7 @@ class Language : public ALabel, public sigc::trackable {
     std::string short_name;
     std::string variant;
     std::string short_description;
+    std::string country_flag() const;
```

Comment:
I think we can use country name from `short_name` in case multi-language layouts.
It also implement this https://github.com/Alexays/Waybar/issues/1245.